release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -214,7 +229,7 @@ For further information about gathering debugging information relating to low-la
 
 [id="ocp-4-14-nw-ipv6-spoke-cluster-support"]
 ==== Support for provisioning IPv6 spoke clusters from dual-stack hub clusters
-With this update, you can provision IPv6 address spoke clusters from dual-stack hub clusters. In a zero touch provisioning (ZTP) environment, the HTTP server on the hub cluster that hosts the boot ISO now listens on both IPv4 and IPv6 networks. The provisioning service also checks the baseboard management controller (BMC) address scheme on the target spoke cluster and provides a matching URL for the installation media. These updates offer the ability to provision single-stack, IPv6 spoke clusters from a dual-stack hub cluster. 
+With this update, you can provision IPv6 address spoke clusters from dual-stack hub clusters. In a zero touch provisioning (ZTP) environment, the HTTP server on the hub cluster that hosts the boot ISO now listens on both IPv4 and IPv6 networks. The provisioning service also checks the baseboard management controller (BMC) address scheme on the target spoke cluster and provides a matching URL for the installation media. These updates offer the ability to provision single-stack, IPv6 spoke clusters from a dual-stack hub cluster.
```

Comment:
seems this is not related to the nmstate UI feature, should we commit this in another PR?

Reply:
It's trailing white space that my editor auto deletes when I save a file so builds don't break. I didn't actually write anything here or add new content.

Reply:
yeah, okay for me, sorry for not reading carefully